src/skills-builder/skills-builder-modal/skillsBuilderModal.scss

```diff
@@ -14,3 +14,7 @@ $breakpoint-medium: 992px;
     top: 1rem;
   }
 }
+
+.chip-max-width {
+  max-width: 16rem;
```

Comment:
This is to fix an issue where the chips were forcing a wider width on the `SelectableBox` components. This will truncate the text if it is too long.

Reply:
It was not logged for the `SelectableBox` component. That was something I picked up on in a meeting with Gabe.

However, as it turns out, this also fixes the issue we were seeing where Cards in the carousel were getting melded together. That [bug was logged here](https://2u-internal.atlassian.net/browse/APER-2466).